include/beast/http/impl/write.ipp

```diff
@@ -29,6 +30,114 @@ namespace http {
 
 namespace detail {
 
+template<class Streambuf, class Body, class Headers>
+void
+write_firstline(Streambuf& streambuf,
+    message_v1<true, Body, Headers> const& msg)
+{
+    write(streambuf, msg.method);
+    write(streambuf, " ");
+    write(streambuf, msg.url);
+    switch(msg.version)
+    {
+    case 10:
+        write(streambuf, " HTTP/1.0\r\n");
+        break;
+    case 11:
+        write(streambuf, " HTTP/1.1\r\n");
+        break;
+    default:
+        write(streambuf, " HTTP/");
+        write(streambuf, msg.version / 10);
+        write(streambuf, ".");
+        write(streambuf, msg.version % 10);
```

Comment:
As HTTP/2 is already out, I have confidence that we are okay. Anyway, I"m going to refactor this code because its not getting coverage (and likely never will, because we will probably never see anything new below HTTP/2 in our lifetime).